src/constants.py

```diff
@@ -0,0 +1,7 @@
+# Copyright 2023 Canonical Ltd.
```

Comment:
IMHO the approach of defining constants in a dedicated file breaks encapsulation. Is there any reason why these are not defined in `charm.py`?

Reply:
I took the idea from [mysql-k8s-operator repo](https://github.com/canonical/mysql-k8s-operator/blob/d83c4f3c6a05d0ec688412f2e497eb9c94a5b603/src/constants.py), it seemed better to separate the common constants out into a separate file so that later when other relations get put into a separate file there will be no cyclic dependency problem.
My intention for the refactor was to allow grounding for each relation to be separated it it's own file to better separate out code for each of the relation in it's own respective file.
Since this refactor should probably go through a spec, I'm fine with reverting this particular change and moving the constants in `charm.py`. What do you think?

Reply:
I'd go with reverting since there's currently no motivation for keeping these constants here. Perhaps in the future it will be a good solution.

Reply:
Done :)